testcontainers/rabbitmq.py

```diff
@@ -0,0 +1,71 @@
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.waiting_utils import wait_container_is_ready
+import os
+import pika
+
+
+class RabbitmqContainer(DockerContainer):
+    RABBITMQ_DEFAULT_USER = os.environ.get("RABBITMQ_DEFAULT_USER", "guest")
+    RABBITMQ_DEFAULT_PASS = os.environ.get("RABBITMQ_DEFAULT_PASS", "guest")
+
+    def __init__(self, image="rabbitmq:latest"):
+        super(RabbitmqContainer, self).__init__(image=image)
+        self.port_to_expose = 5672
+        self.with_exposed_ports(self.port_to_expose)
+        self.queues=list()
+
+    def _configure(self):
+        self.with_env('RABBITMQ_DEFAULT_USER', 'guest')
+        self.with_env('RABBITMQ_DEFAULT_PASS', 'guest')
+
+    @wait_container_is_ready()
+    def get_connection_parameters(self):
+        credentials = pika.PlainCredentials(self.RABBITMQ_DEFAULT_USER, self.RABBITMQ_DEFAULT_PASS)
+        parameters = pika.ConnectionParameters(self.get_container_host_ip(), self.port_to_expose,
+                                               '/', credentials)
+        return parameters
+
+    @wait_container_is_ready()
+    def declare_queue(self, queue):
```

Comment:
This seems to be a duplicate (in terms of signature) of the function in line 48. Should this be removed?

Reply:
according to the logic - this method is supposed to declare multiple queues